Python/Product/AnacondaInstallLauncher/AnacondaInstallLauncher.csproj

```diff
@@ -11,14 +11,14 @@
         <OldToolsVersion>14.0</OldToolsVersion>
       </PropertyGroup>
     </When>
-    <When Condition=" '$(VisualStudioVersion)'=='14.0'  Or '$(TargetVisualStudioVersion)'=='VS140' ">
+    <When Condition=" '$(VisualStudioVersion)'=='16.0'  Or '$(TargetVisualStudioVersion)'=='VS160' ">
```

Comment:
Don't we have a common props file where we can store these conditions?

Reply:
Yes, but unfortunately these ones need to stay in the .csproj or VS can't open the file properly. Maybe they've fixed these bugs in VS by now? But last time I checked they hadn't.

Reply:
(The bug is that they only look in the current file to find these properties and don't resolve through Imports)